dgds_backend/config_data/datasets_access.json

```diff
@@ -1,4 +1,106 @@
 {
+  "chasm_wvh": {
+    "dataService":  {
+      "url": "http://pl-tc012.xtr.deltares.nl:8080/FewsWebServices/rest/digitaledelta/2.0",
+      "name": "",
```

Comment:
Need the data id, or do we not have these available yet?

Reply:
Good one, will ask and update.